src/nvim/eval/vars.c

```diff
@@ -696,7 +696,7 @@ static char *ex_let_one(char *arg, typval_T *const tv, const bool copy, const bo
 
       if (!failed) {
         if (opt_type != gov_string || s != NULL) {
-          char *err = set_option_value(arg, n, s, scope);
+          char *err = set_option_value(arg, (const long)n, s, scope);
```

Comment:
seems weird that "const" needed in a value cast

Reply:
You're right, it's not needed. Thanks.